### PR TITLE
Request to merge work from GSOC 2024: all the tests are finally passing

### PR DESCRIPTION
**Pull request by jguillen15**
Regression test will probably fail because we need to set up the IDT credentials as a GitHub secret in the parent iGEM-Engineering/iGEM-actions repository

### CODE REVIEW

**Comment by jakebeal** [comment]
I have confirmed that the test that is failing here is passing in the branch being merged, and thus should not have a problem once credentials are local.

File: scripts/test/test_files/package.nt
Comment (jakebeal) [inline scripts/test/test_files/package.nt:1]: This looks like an accidental commit, duplicating the one in `views/package.nt`
Please try to remove it and see if that breaks any tests.

**Review by jakebeal — APPROVED** [review]
Approve pending the test removal of package.nt

File: scripts/test/test_files/package.nt
Comment (jguillen15) [reply scripts/test/test_files/package.nt:1]: I double checked and it is not an accidental commit. I use the file for testing purposes in `test_compute_complexity_score.py` on line 31. I took the [package.nt file](https://github.com/iGEM-Engineering/iGEM-RBS-collection/blob/main/views/package.nt) from the [RBS-Collection repository](https://github.com/iGEM-Engineering/iGEM-RBS-collection) and didn't rename it appropiately. I changed the name of the file to `package_RBS_Collection.nt`

**Pull request merged**